pkg/kubelet/status/status_manager.go

```diff
@@ -402,9 +404,17 @@ func (m *manager) updateStatusInternal(pod *v1.Pod, status v1.PodStatus, forceUp
 	}
 
 	normalizeStatus(pod, &status)
+
 	// The intent here is to prevent concurrent updates to a pod's status from
 	// clobbering each other so the phase of a pod progresses monotonically.
-	if isCached && isPodStatusByKubeletEqual(&cachedStatus.status, &status) && !forceUpdate {
+	shouldUpdate := forceUpdate
+	if !shouldUpdate && m.lastUpdateTime != nil {
+		if time.Now().Sub(*m.lastUpdateTime) > 200*time.Second {
```

Comment:
any indicates why 200s is used here?

Reply:
It represents long period of time beyond which pod status must be updated.
I am open to suggestion on the actual value.

Reply:
I am ok with 200s here, but can you please add a comment on what are you doing here and why choose 200s?

Reply:
@dchen1107 
I have added comment for this code.

Reply:
Define it as a constant and explain "why" 200s is chosen. The current comment does not explain the magic number.